shoop_tests/core/test_shops.py

```diff
@@ -20,4 +20,4 @@ def test_shop_wont_be_deleted():
     shop.save()
     img.delete()
 
-    Shop.objects.get(pk=1)
+    Shop.objects.first()
```

Comment:
This change actually makes the test a no-op. Where `.get()` would have raised an ODNE when the object does not exist, this quietly returns None.

Change this to `Shop.objects.get(pk=shop.pk)` instead.